FrontEnd/src/app/src/components/user-edit/user-edit.component.html

```diff
@@ -0,0 +1,97 @@
+<app-drawer>
+  <ng-container main-content>
+    <span class="mainPage">
+      <mat-card class="tarjeta">
+        <mat-card-header>
+          <div class="header">
+            <button mat-icon-button class="back-btn" aria-label="Go back" (click)="homePage()">
+              <mat-icon>chevron_left</mat-icon>
+            </button>
+
+            <div class="avatar-container">
+              <div class="user-avatar">
+                <img class="user-avatar-img" [src]="userImage" alt="User Image">
+              </div>
+            </div>
+
+            <div class="role-container">
+              <mat-button-toggle-group
+                name="role"
+                aria-label="role"
+                [(ngModel)]="user.role.name" required>
+                <mat-button-toggle (click)="adminRole()" value="Admin">Admin</mat-button-toggle>
+                <mat-button-toggle (click)="userRole()" value="User">User</mat-button-toggle>
+              </mat-button-toggle-group>
```

Comment:
si tenes ganas de experimentar y si queres, proba usar una directiva de tipo @for